app/views/system_admin/users/show.html.erb

```diff
@@ -1,69 +1,84 @@
-      <%= @user.last_name %>
-    </dd>
+    <dt class="govuk-summary-list__key">Last name</dt>
+    <dd class="govuk-summary-list__value"><%= @user.last_name %></dd>
   </div>
   <div class="govuk-summary-list__row">
-    <dt class="govuk-summary-list__key">
-      Email
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <%= @user.email %>
-    </dd>
+    <dt class="govuk-summary-list__key">Email</dt>
+    <dd class="govuk-summary-list__value"><%= @user.email %></dd>
   </div>
-  <div class="govuk-summary-list__row lead-school-row">
-    <dt class="govuk-summary-list__key">
-      Lead schools
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <% if @user.lead_schools.any? %>
-        <div class="govuk-!-margin-bottom-3">
-          <% @user.lead_schools.each do |lead_school| %>
-            <%= govuk_link_to("#{lead_school.name} - #{lead_school.urn}", lead_school_path(lead_school), class: "govuk-!-display-block") %>
-          <% end %>
-        </div>
-      <% end %>
-      <%= govuk_link_to("Add lead school", new_user_lead_school_path(@user), class: "govuk-!-display-block add-lead-school-to-user") %>
-    </dd>
-  </div>
-  <div class="govuk-summary-list__row provider-row">
-    <dt class="govuk-summary-list__key">
-      Providers
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <% if @user.providers.any? %>
-        <div class="govuk-!-margin-bottom-3">
-          <% @user.providers.each do |provider| %>
-            <%= govuk_link_to "#{provider.name} - #{provider.code}", provider_path(provider), class: "govuk-!-display-block" %>
-          <% end %>
-        </div>
-      <% end %>
-      <%= govuk_link_to("Add provider", new_user_provider_path(@user), class: "govuk-!-display-block add-provider-to-user") %>
-      </div>
-    </dd>
 </dl>
-  
-<%= govuk_link_to("Delete this user", delete_user_path(@user), class: "link--delete govuk-link--no-visited-state govuk-body") unless @user.system_admin? %>
\ No newline at end of file
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <table id="providers" class="govuk-table">
+      <caption class="govuk-table__caption govuk-table__caption--m">Providers</caption>
+      <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Name</th>
+        <th scope="col" class="govuk-table__header">Remove access</th>
+      </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <% @user.providers.each do |provider| %>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">
+              <%= govuk_link_to provider.name, provider_path(provider), class: 'provider' %>
+            </td>
+            <td class="govuk-table__cell">
+              <%= govuk_link_to "Remove", edit_user_provider_accessions_path(@user, provider), class: 'remove-access' %>
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
```

Comment:
Can we pull out `Providers` as a heading, and only show the table where there are providers? When there are no providers show:

```
<p class="govuk-body govuk-hint">User is not a member of any providers</p>
```


Example from prototype:
<img width="767" alt="Screenshot 2022-09-28 at 15 15 44" src="https://user-images.githubusercontent.com/2204224/192802917-a03dbe48-f279-400c-8c7a-ed07e4986a6c.png">